src/modules/cp/const/requests.const.ts

```diff
@@ -12,8 +12,8 @@ export const CP_ADMIN_REQUESTS_QUERY_PARAM_CONFIG = {
 	status: withDefault(StringParam, RequestStatusAll.ALL),
 	search: withDefault(StringParam, undefined),
 	page: withDefault(NumberParam, 1),
-	sort: withDefault(StringParam, undefined),
-	order: withDefault(SortDirectionParam, undefined),
+	orderProp: withDefault(StringParam, undefined),
+	orderDirection: withDefault(SortDirectionParam, undefined),
```

Comment:
`sort` & `order` staan expliciet in de acceptatiecriteria
https://meemoo.atlassian.net/jira/software/c/projects/ARC/boards/36?modal=detail&selectedIssue=ARC-397

Reply:
Besproken met @andrych, https://meemoo.atlassian.net/browse/ARC-397?focusedCommentId=29094